README.md

```diff
@@ -1,66 +1,55 @@
-     * [Testing](./docs/hipify-clang.md#hipify-clang-testing)
-        + [Linux](./docs/hipify-clang.md#hipify-clang-linux-testing)
-        + [Windows](./docs/hipify-clang.md#hipify-clang-windows-testing)
-- [hipify-perl](./docs/hipify-perl.md#hipify-perl)
-     * [Usage](./docs/hipify-perl.md#hipify-perl-usage)
-     * [Building](./docs/hipify-perl.md#hipify-perl-building)
-- Related: [hipify_torch](https://github.com/ROCmSoftwarePlatform/hipify_torch)
-- [Supported CUDA APIs](./docs/supported_apis.md#supported-cuda-apis)
-- [Documentation](#documentation)
-     * [How to build HIPIFY documentation](#how-to-build-hipify-documentation)
-     * [How to build CUDA2HIP documentation](#how-to-build-cuda2hip-documentation)
-- [Disclaimer](#disclaimer)
+* [hipify-clang](./docs/hipify-clang.md#hipify-clang)
+  * [Dependencies](./docs/hipify-clang.md#hipify-clang-dependencies)
+  * [Usage](./docs/hipify-clang.md#hipify-clang-usage)
+  * [Building](./docs/hipify-clang.md#hipify-clang-building)
+  * [Testing](./docs/hipify-clang.md#hipify-clang-testing)
+    * [Linux](./docs/hipify-clang.md#hipify-clang-linux-testing)
+    * [Windows](./docs/hipify-clang.md#hipify-clang-windows-testing)
+* [hipify-perl](./docs/hipify-perl.md#hipify-perl)
+  * [Usage](./docs/hipify-perl.md#hipify-perl-usage)
+  * [Building](./docs/hipify-perl.md#hipify-perl-building)
+* Related: [hipify_torch](https://github.com/ROCmSoftwarePlatform/hipify_torch)
+* [Supported CUDA APIs](./docs/supported_apis.md#supported-cuda-apis)
+* [Documentation](#documentation)
+  * [How to build HIPIFY documentation](#how-to-build-hipify-documentation)
+  * [How to build CUDA2HIP documentation](#how-to-build-cuda2hip-documentation)
+* [Disclaimer](#disclaimer)
 
 <!-- tocstop -->
 
-## <a name="documentation"></a>Documentation
+## Documentation
 
-Information about HIPIFY and other user topics is found in the [HIPIFY documentation](https://rocmdocs.amd.com/projects/HIPIFY/en/latest/).
+Documentation for HIPIFY is available at
+[[HIPIFY documentation](https://rocmdocs.amd.com/projects/HIPIFY/en/latest/)](https://rocmdocs.amd.com/projects/HIPIFY/en/latest/).
 
-### <a name="how-to-build-HIPIFY-documentation"></a>How to build HIPIFY documentation
+To build our documentation locally, run the following code.
 
-Run the steps below to build the HIPIFY documentation locally.
-
-```
+```bash
 cd docs
 
 pip3 install -r .sphinx/requirements.txt
 
 python3 -m sphinx -T -E -b html -d _build/doctrees -D language=en . _build/html
 ```
 
-### <a name="how-to-build-CUDA2HIP-documentation"></a>How to build CUDA2HIP documentation
-
-Run the following `hipify-clang` command to build the documentation for CUDA APIs supported by HIP (CUDA2HIP) locally.
+To build `CUDA2HIP` (CUDA APIs supported by HIP) documentation, run the following `hipify-clang`
+command. This builds the same content as
+[Supported CUDA APIs](./docs/supported_apis.md#supported-cuda-apis).
```

Comment:
Actually no, to build the same CUDA2HIP a few more command lines are needed:
hipify-clang --md --doc-format=full  --doc-roc=joint
hipify-clang --md --doc-format=full  --doc-roc=separate